indexer-app/src/cmr/indexer/data/index_set.clj

```diff
@@ -416,6 +416,9 @@
   (merge
     {:concept-id (m/stored m/string-field-mapping)
 
+     :native-id m/string-field-mapping
+     :native-id.lowercase m/string-field-mapping
```

Comment:
If we are not aggregating or sorting on native-id we should be fine without using doc-values though I think Mark is right that all new fields should use doc-values in any case (you don't have to name the field native-id-doc-values, just set up the mapping). In later versions of Elasticsearch they have started to default all fields to doc-values.

Here's some documentation - https://www.elastic.co/guide/en/elasticsearch/guide/1.x/doc-values.html